unfetter-discover/unfetter
# Pass proxy as buildarg to Docker builds

Requires unfetter-discover/unfetter#____

Proxy defined in hostvars files will be honored and passed as `buildarg` into Dockerfile builds.

### To test:
1. Pull this branch and branches specified above
2. Ensure that /unfetter/ansible/backup directory is deleted, and no proxy variables are set in the OS environment
3. Define a dev.yml file in ansible/host_vars, and include an `https_proxy_url` variable, similar to this:
     https_proxy_url: "http://my.proxy.com:80"
4. Perform a dev build:  `ansible-playbook deploy-dev.yml`
5. No timeout errors should occur in any Docker image builds
